### DISCUSSION TITLE
Proposal: Line Renderer Node

### DISCUSSION THREAD

**Discussion by local-interloper** [godotengine/godot-proposals#8670] [3D]
As of now, there's no straightforward/"out of the box" way of rendering lines in 3D (none that I am aware of). I propose adding an equivalent to Unity's line rendering component where a component can tile/stretch a texture across two or more points in space. This would make it easy to draw stuff like bullet traces, lasers, ropes, etc...

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/6151.
